ner/pom.xml

```diff
@@ -89,6 +89,11 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.4</version>
         </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-ner</artifactId>
+            <version>3.0.58</version>
+        </dependency>
```

Comment:
I didn't see this. I only committed 2 files, but it seems that I missed one. I think you can ignore this. Sorry for that.